front-end/src/host/HostApp.tsx

```diff
@@ -195,40 +195,38 @@ export default function HostApp(props: IHostProps) {
   return (
     <div className="scroll">
       <PlayAudio src={theme} loop={true} />
-      <div className="musicButton">
-        <IconButton onClick={() => muteMusic(muted)}>
-          <img className="musicIcon" src={muted ? musicOff : musicOn} />
-        </IconButton>
-      </div>
-      <div className="hostFormat">
-        <img className="logohost" src={logo} />
-        {getElementForState(gameState, settingsState)}
+      <div className="banner">
+        <div className="musicButton">
+          <IconButton onClick={() => muteMusic(muted)}>
+            <img className="musicIcon" src={muted ? musicOff : musicOn} />
+          </IconButton>
+        </div>
+        <div className="hostFormat">
+          <img className="logohost" src={logo} />
+        </div>
       </div>
+      {gameState == "lobby" ? (
```

Comment:
since the only thing different here is the `className` prop, you could refactor this a bit... basically, this part would always be there:

```html
<div className="">
  {getElementForState(gameState, settingsState)}
</div>
```

but the className could be conditional:

```html
<div className={gameState == "lobby" ? "hostFormat1" : "hostFormat`"}>
<!-- etc --->
```

Also, may be a good idea to come up with different names for these classes - I'm not really sure what the difference is between `hostFormat` and `hostFormat1`, so it would be good to clarify that through naming!